install.sh

```diff
@@ -16,6 +16,7 @@ apt-get install -y python-scipy
 apt-get install libcairo2-dev
 apt-get install libffi-dev
 pip install cairocffi
+pip install coverage
```

Comment:
Please double check the command on a server machine.

Reply:
Sure, will do that on Monday. I am not on campus.